config/prow-staging/core/config.yaml

```diff
@@ -108,11 +108,21 @@ tide:
     labels:
     - lgtm
     - approved
-    missingLabels:
+    - "cla: yes"
+    missingLabels: &knative_tide_missing_labels
     - do-not-merge/hold
     - do-not-merge/work-in-progress
     - do-not-merge/invalid-owners-file
     - needs-rebase
+  - repos:
+    # Allow PRs for test-infra to be automatically merged without manual approval.
+    - "knative-prow-robot/test-infra"
+    labels:
+    - "cla: yes"
+    - auto-merge
+    missingLabels: *knative_tide_missing_labels
+    author: knative-prow-robot
```

Comment:
I think it's not, but still want to double check, this is not the same bot creating auto images bump , right?

Reply:
No, that bot's name is `knative-prow-updater-robot`.